docs/contributors.md

```diff
@@ -1,40 +1,113 @@
+
+- CI will start automatically with some checks. Wait until the end of the execution and make sure that all checks passed successfully. If some checks fail, you can open them one by one, try to find the reason for failing and make changes to your code to resolve the problem
+> Sometimes you may need to update the installers you can do this with the `npm run update-installers` command
+- Maintainers will review your pull request
+- If a maintainer requests changes, first of all, try to think about his request critically and only after that implement and request another review
+- If your PR gets accepted, it will soon be merged into the main branch. But your contribution will take effect only after the release of a new version of the action
+> Sometimes maintainers reject pull requests and that's ok! Usually, along with rejection, we supply the reason for it. Nonetheless, we still really appreciate you taking the time to do it, and we don't take that lightly :heart:
+
+## Provide support on issues
+
+Helping out other users with their questions is an awesome way of contributing to any community. It's not uncommon for most of the issues on open source projects to be support-related questions by users trying to understand something they ran into or find their way around a known bug.
+
+**To help other folks out with their questions:**
+
+- Go to the [issue tracker](https://github.com/actions/setup-dotnet/issues)
+- Read through the list until you find something that you're familiar enough with to answer to
+- Respond to the issue with whatever details are needed to clarify the question, or get more details about what's going on
+- Once the discussion wraps up and things are clarified, ask the original issue filer (or a maintainer) to close it for you
+ 
+*Some notes on picking up support issues:*
+
+- Avoid responding to issues you don't know you can answer accurately
+- Try to refer to past issues with accepted answers as much as possible. Link to them from your replies
+- Be kind and patient with users. Often, folks who have run into confusing things might be upset or impatient. This is natural. If you feel uncomfortable in conversation with them, it's better to stay away or withdraw from the issue.
+
+ > If some user is violating our code of conduct [standards](https://github.com/actions/setup-dotnet/blob/main/CODE_OF_CONDUCT.md#our-standards), refer to the [Enforcement](https://github.com/actions/setup-dotnet/blob/main/CODE_OF_CONDUCT.md#enforcement) section of the Code of Conduct to resolve the conflict
+ 
+## Review pull requests
+ 
+Another great way to contribute is pull request reviews. Please, be extra kind: people who submit code/doc contributions are putting themselves in a pretty vulnerable position, and have put time and care into what they've done (even if that's not obvious to you!) Please, always respond with respect, and be understanding, but don't feel like you need to sacrifice your standards for their sake, either.
```

Comment:
Another great way to contribute is pull request reviews. => Another great way to contribute is to review pull request.